### DISCUSSION TITLE
Hacktoberfest?

### DISCUSSION THREAD

**Discussion by ThatXliner** [python-poetry/poetry#6675] [Q&A] [answered]
Are you guys going to join hacktoberfest or are there specific reasons not to?

**Comment by neersighted** [comment]
I'm cautiously interested personally, but consensus from other project regulars would be wonderful.

Basically, in previous years issues/PRs languished as there wasn't enough review bandwidth. I'd like to avoid 'false advertising' or similar if we don't have enough review bandwidth.

**Comment by neersighted** [comment, marked as answer]
Discussed it with some other members of the team, and the conclusion is that we're opting out this year. We've got a lot of high quality PRs open with a not a lot of review bandwidth, and even trivial changes to Poetry can often be non-trivial to review, as there are a lot of stumbling blocks for new contributors. We've got maybe 5 'simple' PRs open right now that have taken a decent amount of effort/cycles and will likely require more before they can be merged.
